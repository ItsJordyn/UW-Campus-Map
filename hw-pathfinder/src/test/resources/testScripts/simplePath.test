# Find the shortest path with Dijkstra's algorithm

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 11.0
AddEdge simpleGraph A D 6.0
AddEdge simpleGraph A C 2.0
AddEdge simpleGraph C D 3.0
AddEdge simpleGraph D B 3.0

FindPath simpleGraph A B