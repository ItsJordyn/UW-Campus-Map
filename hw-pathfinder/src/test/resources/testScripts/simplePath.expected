# Find the shortest path with Dijkstra's algorithm

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 11.000 from A to B in simpleGraph
added edge 6.000 from A to D in simpleGraph
added edge 2.000 from A to C in simpleGraph
added edge 3.000 from C to D in simpleGraph
added edge 3.000 from D to B in simpleGraph

path from A to B:
A to C with weight 2.000
C to D with weight 3.000
D to B with weight 3.000
total cost: 8.000